cmd/kogito/command/deploy/deploy_service_test.go

```diff
@@ -136,7 +136,7 @@ my.nice.property=socool
 	exists, err := kubernetes.ResourceC(ctx.GetClient()).Fetch(cm)
 	assert.NoError(t, err)
 	assert.True(t, exists)
-	assert.Contains(t, cm.Data[kogitoservice.ConfigMapApplicationPropertyKey], "quarkus.log.level")
+	assert.Contains(t, cm.Data[converter.ConfigMapApplicationPropertyKey], "quarkus.log.level")
```

Comment:
maybe compare the full value here? property=value?

Reply:
done